org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/tool/JrtFileSystem.java

```diff
@@ -55,7 +56,7 @@ public void initialize() throws IOException {
 		// initialize packages
 		this.modulePathMap = new HashMap<>();
 		if (this.file.exists()) {
-			this.jrtfs = JRTUtil.getJrtFileSystem(this.file.getAbsolutePath());
+			this.jrtfs = JRTUtil.getJrtFileSystem(this.file.toPath());
```

Comment:
Btw. I noticed that this field is overwritten in the `JrtFileSystemWithOlderRelease` sub-class with the CtSystem, but `modRoot` isn't.
I assume that's all intentional? (I'm not very familiar with that code, I just wanted to make sure all is as it should be:) )